core/src/main/java/io/snappydata/impl/SnappyHiveCatalog.java

```diff
@@ -328,7 +328,7 @@ public Object call() throws Exception {
           Object value = parameters.get(StoreUtils.GEM_INDEXED_TABLE());
           String baseTable = value != null ? value.toString() : "";
           String dmls = JdbcExtendedUtils.
-              getInsertOrPutString(fullyQualifiedName, schema, false);
+              getInsertOrPutString(fullyQualifiedName, schema, false, false);
```

Comment:
getInsertOrPutString does not need to escape quotes as the insert/put formed here isn't used?